spec/worker-helpers-spec.js

```diff
@@ -7,42 +7,58 @@ import * as Path from 'path'
 describe('Worker Helpers', () => {
   describe('getESLintInstance && getESLintFromDirectory', () => {
     it('tries to find a local eslint', () => {
-      const eslint = Helpers.getESLintInstance(getFixturesPath('local-eslint'), {})
+      const fileDir = getFixturesPath('local-eslint')
+      const eslint = Helpers.getESLintInstance(fileDir, '', {})
       expect(eslint).toBe('located')
     })
-    it('cries if local eslint is not found', () => {
-      expect(() => {
-        Helpers.getESLintInstance(getFixturesPath('files', {}))
-      }).toThrow()
```

Comment:
This was actually throwing the wrong error: `Can't read property useGlobalEslint of undefined` because the `{}` was not being correctly passed in to `getESLintInstance`.

I decided to remove this test completely because it will always find our own installation of eslint (the one in `linter-eslint/node_modules`).